README.md

```diff
@@ -8,7 +8,7 @@ See the JavaScript toolkit running in the [MEI Viewer](http://www.verovio.org/me
 
 ![Choice interaction](https://raw.githubusercontent.com/rism-ch/verovio/gh-pages/movies/reflow.gif)
 
-Verovio can also render Plain and Easy (PAE) code and DARMS code and has a basic MusicXML import function. It is designed as a tool usable as a one-step conversion tool and it is written in pure standard C++ with no dependencies on third-party frameworks and libraries.
```

Comment:
Looking at this: I would prefer to keep `with no dependencies on third-party frameworks and libraries.`

Reply:
But wouldn't this be incorrect? We use JSON++, pugixml, Midifile, etc. 
How about 
> with no dependencies on external libraries.